src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputOperatorWrap.java

```diff
@@ -8,24 +8,25 @@ class InputOperatorWrap
 {
     void test()
     {
-        /*warn*/int x = 1 +
-                /*warn*/ 2 -
+        int x = 1 + // violation '.* should be on a new line.'
```

Comment:
.* Is wild card for any symbols of any length. Can we be more specific ?

Reply:
Ummmm these types of symbols causes mismatching in comparing violation messages so to prevent them I use `.*`. 
We can do something like:

// violation ''.*' should be on a new line.'

I don't any other way to resolve the mismatching

Reply:
Nvm, reformatted the violation message as you suggested here https://github.com/checkstyle/checkstyle/pull/15190#discussion_r1664598643